tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -295,6 +295,7 @@ cc_library(
         "@llvm//:object",
         "@llvm//:support",
         "@llvm//:target",
+        "@llvm//:polly",
```

Comment:
@sanjoy @tobig Thank you for this discussion. I am working on getting the benchmarking complete ASAP.